Code/Framework/AzToolsFramework/AzToolsFramework/API/ComponentModeCollectionInterface.h

```diff
@@ -22,11 +22,12 @@ namespace AzToolsFramework
 
         virtual ~ComponentModeCollectionInterface() = default;
 
-        //! Retrieves the list of all Component types (usually one).
+        //! Retrieves the list of all component types (usually one).
         //! @note If called outside of component mode, an empty vector will be returned.
         virtual AZStd::vector<AZ::Uuid> GetComponentTypes() const = 0;
 
-        //! Calls the handler function for each ComponentMode that is currently active.
-        virtual void EnumerateActiveComponents(AZStd::function<void(const AZ::EntityComponentIdPair&, const AZ::Uuid&)> handler) const = 0;
+        //! Calls the handler function for each component mode that is currently active.
+        using ActiveComponentModeCB = AZStd::function<void(const AZ::EntityComponentIdPair&, const AZ::Uuid&)>;
```

Comment:
standards/nit: This should ideally be `Cb` not `CB` to be consistent with the coding standards (see https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#abbreviations)

Reply:
The rest of the codebase uses `CB` so I'd like to conform to that right now. I see your point though :)